go/client/cmd_wallet_balances.go

```diff
@@ -98,7 +98,14 @@ func (c *cmdWalletBalances) runForUser(cli stellar1.LocalClient) error {
 				if acc.ExchangeRate != nil {
 					localAmount, err := stellar.ConvertXLMToOutside(balance.Amount, *acc.ExchangeRate)
 					if err == nil {
-						localAmountStr = fmt.Sprintf(" (~%s %s)", localAmount, string(acc.ExchangeRate.Currency))
+						ret, err := cli.FormatCurrencyString(context.Background(), stellar1.FormatCurrencyStringArg{
+							Money: localAmount,
+							Code:  acc.ExchangeRate.Currency,
+						})
+						if err != nil {
+							c.G().Log.Warning("Unable to format amount in: %s", err)
+						}
+						localAmountStr = fmt.Sprintf(" (%s ~%s)", string(acc.ExchangeRate.Currency), ret)
```

Comment:
This seems like a great way to make sure the system is working. But I don't think it looks better with both the code and the symbol. Just the code before seemed fine, especially given there's a squiggle. This version uses `ret` even if it errored out.